thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js

```diff
@@ -116,6 +113,18 @@ export default Ember.Component.extend({
     return this.get('status') === 'NO_FEEDBACK';
   }),
 
+  isHidden: Ember.computed(
+    'isNeedFeedback',
```

Comment:
nit: `needsFeedback` might be more correct (grammatically)

Reply:
yeah that's a habit of prefixing any boolean return type with 'is'. I'll rename it